README.md

```diff
@@ -34,12 +34,21 @@ Extend @ni in the ESLint configuration.
 
 ### TypeScript
 
-Extend @ni/eslint-config/typescript in the ESLint configuration. Configure the @typescript-eslint plugin and the project's TypeScript configuration.
+Extend `@ni/eslint-config/typescript` in the default ESLint configuration. The default `.eslintrc.json` used for development should only extend TypeScript rules and not the TypeScript rules requiring type checking.
 
 ```json
 {
     "extends": "@ni/eslint-config/typescript",
-    "plugins": ["@typescript-eslint"],
```

Comment:
I removed the plugins config from the root README as it is already in the [typescript-eslint base](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/base.ts#L8) and it seems to work without it.

Let me know if we should add it back. The [getting started docs](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md#configuration) seem to indicate configuring both `plugins` and `parser` is needed but it does not seem that way locally and I think it's because they are configured in base.

Reply:
I didn't have any problems when I removed `plugins` from my config files.

Reply:
Was also tested by nicolas in web portal so marking resolved